src/chart/generateCategoricalChart.tsx

```diff
@@ -2040,6 +2041,24 @@ export const generateCategoricalChart = ({
       );
     };
 
+    static renderActiveBar = (option: any, props: any): React.ReactElement => {
```

Comment:
We might also want to start moving away from static functions like this @nikolasrieble  thoughts on moving this to `Bar` or a new `BarUtils`? It doesn't really use anything within the `generateCategoricalChart` closure. 

We could refactor all of them in a separate PR I guess

Reply:
Moved this to a `BarUtils` file.